app/models/pregnancy.rb

```diff
@@ -67,4 +68,34 @@ def recent_calls
   def old_calls
     calls.order('created_at DESC').offset(10)
   end
+
+  def contact_made?
+    self.calls.each do |call|
```

Comment:
If memory serves, you might be able to omit the `self` reference here -- I think Ruby is smart enough to know to check calls on the given object.